lib/mws/fulfillment_inbound_shipment/client.rb

```diff
@@ -7,7 +7,8 @@ module FulfillmentInboundShipment
     # also request lists of inbound shipments or inbound shipment items based on
     # criteria that you specify.
     class Client < ::Peddler::Client
-      path '/FulfillmentInboundShipment/2010-10-01'
+      version '2010-10-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.